docsrc/Connecting_and_queries.rst

```diff
@@ -38,17 +37,20 @@ To get started, follow the steps below:
 
 	A connection requires the following parameters:
 
-	+------------------------------------+-------------------------------------------------------------------+
-	| ``username``                       |  The email address associated with your Firebolt user.            |
-	+------------------------------------+-------------------------------------------------------------------+
-	| ``password``                       |  The password used for connecting to Firebolt.                    |
-	+------------------------------------+-------------------------------------------------------------------+
-	| ``database``                       |  The name of the database you would like to connect to.           |
-	+------------------------------------+-------------------------------------------------------------------+
-	| ``engine_name`` or ``engine_url``  |  The name or URL of the engine to use for SQL queries.            |
-	|                                    |                                                                   |
-	|                                    |	If the engine is not specified, your default engine is used.     |
-	+------------------------------------+-------------------------------------------------------------------+
+	+------------------------------------+-----------------------------------------------------------------------------------------------------+
+	| ``auth``                           |  Auth object, containing your credentials. See :ref:`Auth <firebolt.client:auth>` for more details. |
+	+------------------------------------+-----------------------------------------------------------------------------------------------------+
+	| ``database``                       |  Optional. The name of the database you would like to connect to.                                   |
+	+------------------------------------+-----------------------------------------------------------------------------------------------------+
+	| ``engine_name``                    |  Optional. The name of the engine to use for SQL queries.                                           |
+	+------------------------------------+-----------------------------------------------------------------------------------------------------+
+
+	.. note::
+		If you specify ``engine_name`` but not the ``database`` Python SDK will automatically resolve the database for you behind the scenes.
+
+		If an ``engine_name`` is not specified the SDK will connect to a system engine. In this case, if no ``database`` is specified for a system engine
```

Comment:
Can we say something like `If an ``engine_name`` is not specified the SDK connection will not be bound to any engine`